extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -112,7 +106,6 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverRubiconDisplayPrebidCountries';
 		$vars[] = 'wgAdDriverRubiconPrebidCountries';
 		$vars[] = 'wgAdDriverRubiconDfpCountries';
-		$vars[] = 'wgAdDriverRubiconVideoInFeaturedVideoCountries';
```

Comment:
We'll need to postpone this PR till https://github.com/Wikia/mobile-wiki/pull/1466 is released